docs/content/preview/integrations/hashicorp-vault.md

```diff
@@ -67,6 +67,10 @@ Install the YSQL plugin for Hashicorp Vault as follows:
     go build -o <build dir>/ysql-plugin cmd/ysql-plugin/main.go
     ```
 
+Alternatively, you can download the binary directly from GitHub:
+
+- Pre-built binary can be found at the [releases page](https://github.com/yugabyte/hashicorp-vault-ysql-plugin/releases), download, unzip and place the binary in <build dir>.
```

Comment:
```suggestion
- Pre-built binary can be found on the [releases page](https://github.com/yugabyte/hashicorp-vault-ysql-plugin/releases). Download, unzip the file and place the binary in <build dir>.
```

Reply:
line # 32:
`- Add go to the path as follows:` -> `- Add Go to the path as follows:`

Reply:
Download, unzip the file, and place the binary in <build dir>.

Reply:
@HarshDaryani896 LGTM with the suggestions. Please make the changes and merge.